dcpy/connectors/edm/publishing.py

```diff
@@ -1,32 +1,34 @@
-import os
-from pathlib import Path
-from urllib.parse import urlencode, urljoin
-import pandas as pd
-import geopandas as gpd
+from botocore.exceptions import ClientError
 from datetime import datetime
+import geopandas as gpd
+from io import BytesIO
+import json
+import pandas as pd
+from pathlib import Path
 import pytz
 import re
-from io import BytesIO
-from zipfile import ZipFile
 import typer
 from typing import Callable, TypeVar
+from urllib.parse import urlencode, urljoin
 import yaml
-import json
-
-from dcpy.utils import s3, git, versions
-from dcpy.utils.logging import logger
+from zipfile import ZipFile
 
+from dcpy.configuration import PUBLISHING_BUCKET, CI, BUILD_NAME
 from dcpy.models.connectors.edm.publishing import (
     ProductKey,
     PublishKey,
     DraftKey,
     BuildKey,
 )
 from dcpy.models.lifecycle.builds import BuildMetadata
+from dcpy.utils import s3, git, versions
+from dcpy.utils.logging import logger
 
-from botocore.exceptions import ClientError
+assert (
+    PUBLISHING_BUCKET
+), "'PUBLISHING_BUCKET' must be defined to use edm.publishing connector"
+BUCKET = PUBLISHING_BUCKET
```

Comment:
mypy weirdness - just asserting it doesn't resolve the type (from `Optional`), but asserting and binding to a new variable does